designs/dragonphy_top/constraints/gen_constraints.py

```diff
@@ -86,7 +151,16 @@
 
 set_clock_groups -asynchronous \\
     -group {{ clk_jtag }} \\
-    -group {{ clk_prbsgen }} \\
+    -group {{ \\
```

Comment:
Inter-clock timing of TX clocks is analyzed, but their timing with respect to JTAG and ADC clocks is not.